tags/lib/mixinextras.ytag

```diff
@@ -0,0 +1,9 @@
+type: text
+
+---
+
+Mixin Extras is a small companion library for the Mixin system, offering better ways to use Mixins.
+It is highly recommended to use Mixin Extras as it offers features such as `ModifyExpressionValue` which is a better alternative to `Redirect`
```

Comment:
Once again, not necessarily a better alternative, but definitely a more compatible alternative depending on the use case.

Reply:
Rephrased it to recommend it over "highly recommending" and avoid calling the features as "better"